docs/reference/ilm/actions/ilm-searchable-snapshot.asciidoc

```diff
@@ -14,8 +14,8 @@ index>>. If the original index is part of a
 the data stream.
 
 IMPORTANT: If the `searchable_snapshot` action is used in the hot phase the
-subsequent phases cannot define any of the `shrink`, `forcemerge`, `freeze` or
-`searchable_snapshot` (also available in the cold and frozen phases) actions.
+subsequent phases cannot define any of the `shrink`, `forcemerge` or `freeze`
+actions.
```

Comment:
Nit: We tend to use an oxford comma. I'd also use `include` instead of `define`.
```suggestion
subsequent phases cannot include the `shrink`, `forcemerge`, or `freeze`
actions.
```